packages/synapse-interface/constants/tokens/bridgeable.ts

```diff
@@ -1173,3 +1178,41 @@ export const WETH = new Token({
   priorityRank: 3,
   routeSymbol: 'WETH',
 })
+
+export const CRVUSDC = new Token({
+  visibilityRank: 101,
+  addresses: {
+    [CHAINS.BASE.id]: '0x417Ac0e078398C154EdFadD9Ef675d30Be60Af93',
+  },
+  decimals: {
+    [CHAINS.BASE.id]: 18,
+  },
+  swapExceptions: {},
+  symbol: 'crvUSD',
+  name: 'Curve.fi USD',
+  logo: crvusdLogo,
+  swapableType: 'USD',
+  swapableOn: [],
+  color: 'yellow',
+  priorityRank: 1,
+  routeSymbol: 'crvUSD',
+})
+
+export const USDBC = new Token({
+  visibilityRank: 101,
+  addresses: {
+    [CHAINS.BASE.id]: '0x417Ac0e078398C154EdFadD9Ef675d30Be60Af93',
```

Comment:
Here's the correct address (used to be crvUSD)
```suggestion
    [CHAINS.BASE.id]: '0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA',
```